postcss.config.js

```diff
@@ -2,18 +2,18 @@ module.exports = {
 	plugins: [
 		require( '@wordpress/postcss-themes' )( {
 			defaults: {
-				primary: '#0085ba',
+				primary: '#007cba',
```

Comment:
There are some hardcoded values in the CSS currently using `#007cba`, is this a coincidence or would it be better to switch those to using the theme values? Thoughts?

Reply:
Good catch! We should use the theme values whenever possible to avoid regressions and inconsistencies.

Reply:
A quick search of the code base shows on these references to these colours. https://github.com/ampproject/amp-wp/search?q=0085ba&unscoped_q=0085ba

Where is the issue here?

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3494/commits/adf9cbe29c27f6dab15e4e54d64d152213637047